api/database/helpers/ApiEnums.php

```diff
@@ -251,4 +251,18 @@ public static function poolAdvertisementSecurity() : array
         ];
     }
 
+    /**
+     * Citizenship
+     */
+    const CITIZENSHIP_CITIZEN = 'CITIZEN';
+    const CITIZENSHIP_PR = 'PERMANENT_RESIDENT';
+    const CITIZENSHIP_OTHER = 'OTHER';
+    public static function citizenshipArray() : array
```

Comment:
The return type `array` as a suffix is unnecessary.  I suggest `citizenshipStatuses` to match your schema enum.

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/3564/commits/f959c93ccef78f88438a6cf750f4d06689028e52